### DISCUSSION TITLE
在 Thunderbird 中使用 Outlook 日历

### DISCUSSION THREAD

**Discussion by giscus** [kxxt/kxxt-website#133] [Thread]
# 在 Thunderbird 中使用 Outlook 日历

因为 Outlook 的日历并不走 CalDav 协议，所以没办法直接和 Thunderbird 集成，鉴于目前网络上此方面的教程稀少，遂撰写此文。 打开 Thunderbird 的扩展插件页面，安装 TbSync 和 Provider for Exchange…

https://www.kxxt.dev/blog/outlook-calendar-with-thunderbird/

**Comment by Anz-03** [comment]
thanks

**Comment by szw0407** [comment]
今天才试了一下，TbSync已经支持Office 365账户的2FA了，路过的可以放心用了。

某种程度上，GNOME生态有些过于封闭了，而且很多功能跟不上，比如Microsoft账户登录支持得很勉强（仅限邮件）

**Comment by alrcatraz** [comment]
添加账户时显示找不到账户，请教一下可能是怎么回事？
有时候也会显示没法授权，但我使用的确实是正确的邮箱和密码

**Reply by alrcatraz** [reply]
错误类型好像都是401
